plugins/module_utils/kibana.py

```diff
@@ -413,7 +413,7 @@ def create_pkg_policy(self,pkg_policy_name, pkg_policy_desc, agent_policy_id, in
             for policy_input in policy_template['inputs']:
               inputs_body_entry = {}
               inputs_body_entry['policy_template'] = policy_template['name']
-              inputs_body_entry['enabled'] = True
+              inputs_body_entry['enabled'] = False
```

Comment:
Why the change here?

Reply:
I know we talked about it, but to make a very long story short Elastic sucks at providing default settings for a pkg policy via the REST API.  I am able to create something to feed into the create from the integration, but the newest System Package Policy settings provided via the API were invalid so with a default of True Elastic tries to turn the stream on and it fails so the create fails.  Setting the initial create to False bypasses the failure. Right after that the Expedient default settings are applied so whatever is applied on the initial create doesn't matter anyway.